src/game/client/gui/gamewindowmanager.cpp

```diff
@@ -106,3 +140,2599 @@ void GameWindowManager::Unlink_Child_Window(GameWindow *window)
+                        break;
+                    }
+                }
+            }
+        }
+
+        if (window == nullptr) {
+            for (window = m_windowList; window != nullptr; window = window->m_next) {
+                if ((window->m_status & WIN_STATUS_BELOW) != 0 && (window->m_status & WIN_STATUS_HIDDEN) == 0
+                    && mouse_pos->x >= window->m_region.lo.x && mouse_pos->x <= window->m_region.hi.x
+                    && mouse_pos->y >= window->m_region.lo.y && mouse_pos->y <= window->m_region.hi.y) {
+                    GameWindow *window5 = window->Win_Point_In_Any_Child(mouse_pos->x, mouse_pos->y, true, true);
+
+                    if (window2 == nullptr
+                        && (window5->m_tooltip != nullptr && window5->m_instData.Get_Tooltip_Text_Length() != 0)) {
+                        window2 = window5;
+                    }
+
+                    if ((window->m_status & WIN_STATUS_ENABLED) != 0) {
+                        window = window->Win_Point_In_Child(mouse_pos->x, mouse_pos->y, false, false);
+                        break;
+                    }
+                }
+            }
+        }
+    }
+
+    if (window != nullptr && (window->m_status & WIN_STATUS_NO_INPUT) != 0) {
+        if (window->Win_Get_Parent() != nullptr
+            && (window->Win_Get_Parent()->Win_Get_Instance_Data()->Get_Style() & GWS_COMBO_BOX) != 0) {
+            window = window->Win_Get_Parent();
+        } else {
+            window = nullptr;
+        }
+    }
+
+    if (window != nullptr && (s_sendMousePosMessages || msg != GWM_MOUSE_POS)) {
+        GameWindow *window6 = window;
+        GameWindow *window7 = m_loneWindow;
+
+        do {
+            if (Win_Send_Input_Msg(window6, msg, packed_mouse_coords, 0) != MSG_IGNORED) {
+                break;
+            }
+
+            window6 = window6->m_parent;
+        } while (window6 != nullptr);
+
+        if (m_loneWindow != nullptr && m_loneWindow == window7
+            && (msg == GWM_LEFT_UP || msg == GWM_MIDDLE_UP || msg == GWM_RIGHT_UP || window6 != nullptr)
+            && !m_loneWindow->Win_Is_Child(window6)) {
+            Win_Set_Lone_Window(nullptr);
+        }
+
+        if (window6 != nullptr) {
+            if (msg == GWM_LEFT_DOWN) {
+                m_grabWindow = window6;
+            }
+
+            ret = WIN_INPUT_USED;
+        }
+    }
+
+    if (window2 == nullptr && !Is_Hidden(window)) {
+        window2 = window;
+    }
+
+    if (window2 != nullptr) {
+        if (window2->m_tooltip != nullptr) {
+            window2->m_tooltip(window2, &window2->m_instData, packed_mouse_coords);
+        } else if (window2->m_instData.Get_Tooltip_Text_Length() != 0) {
+            g_theMouse->Set_Cursor_Tooltip(
+                window2->m_instData.Get_Tooltip_Text(), window2->m_instData.m_tooltipDelay, 0, 1.0f);
+        }
+    } else {
+        b = true;
```

Comment:
This bool is set here but never used. Mistake?

Reply:
Seems to be unused from what I can see.